app/components/provider_interface/user_permission_summary_component.html.erb

```diff
@@ -0,0 +1,109 @@
+<%= tag.dl(class: 'govuk-summary-list') do %>
+  <%= tag.div(class: 'govuk-summary-list__row') do %>
+    <dt class="govuk-summary-list__key">
+      Manage users
+    </dt>
+    <dd class="govuk-summary-list__value">
+      <%= can_manage_users_y_n %>
+    </dd>
+    <% if can_change_permissions? %>
+      <dd class="govuk-summary-list__actions">
+        <%= govuk_link_to(provider_interface_provider_user_edit_permissions_path(provider_user, provider), class: 'govuk-!-display-none-print') do %>
+          Change
+          <span class="govuk-visually-hidden">Change</span>
+        <% end %>
+      </dd>
+    <% end %>
+  <% end %>
```

Comment:
It would be nice to iterate over the possible permissions `ProviderPermissions::VALID_PERMISSIONS` and render this block

Reply:
Because the organisation related permissions need to render another component, it's not as simple as that, and I didn't want to use metaprogramming as I find that makes components a lot harder to read.

Reply:
Something like:
```
ProviderPermissions::VALID_PERMISSIONS.each do |permission|
  <general stuff>...
  <% if ProviderRelationshipPermissions::PERMISSIONS.include?(permission) %>
    <%= render SpecialComponent %>
  <% end %>
end
```